include/cantera/zeroD/ReactorBase.h

```diff
@@ -78,12 +89,17 @@ class ReactorBase
         m_vol = vol;
     }
 
+    //! @deprecated To be removed after %Cantera 3.1. Superseded by setSolution.
+    void insert(shared_ptr<Solution> sol);
+
     //! Specify the mixture contained in the reactor. Note that a pointer to
     //! this substance is stored, and as the integration proceeds, the state of
     //! the substance is modified.
+    //! @todo: make protected (should only be used internally)
```

Comment:
I'm a little hesitant, as `@since` implies that it is already done; also, the nice thing about `@todo` is that it can be easily searched for and is recognized by doxygen. I'm combining what you suggest with what I had already.